lib/takeaway.rb

```diff
@@ -0,0 +1,47 @@
+require_relative ('text')
+
+
+class Takeaway
```

Comment:
If you had to describe the purpose of this object's behaviour - what would it be?